src/main/proto/cosmas.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+

```

Comment:
I think you need to specify at least `java_package` and `java_outer_classname` options ([see the docs](https://developers.google.com/protocol-buffers/docs/javatutorial))

Java package is `com.bardsoftware.papeeria.backend.cosmas`, outer class name `CosmasProto`

Reply:
done